File: Mathematics/1028 - Collectable Cards/1028.c
#include <stdio.h>

int main()
{
    int N, F1, F2, MDC;
    (scanf("%i", &N) > 0);
    while (N > 0)
    {
        scanf("%i %i", &F1, &F2);
        do
        {
            MDC = F1 % F2;
            F1 = F2;
            F2 = MDC;
        } while (MDC != 0);
        printf("%i\n", F1);
        N--;
    }
    return 0;
}

File: Mathematics/1028 - Collectable Cards/1028.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int N, F1, F2, MDC;
        N = int.Parse(Console.ReadLine());
        while (N > 0)
        {
            string[] inputs = Console.ReadLine().Split(' ');
            F1 = int.Parse(inputs[0]);
            F2 = int.Parse(inputs[1]);
            do
            {
                MDC = F1 % F2;
                F1 = F2;
                F2 = MDC;
            } while (MDC != 0);
            Console.WriteLine(F1);
            N--;
        }
    }
}